admin/google_search_console/class-crawl-issue-fetch.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * @package WPSEO\Admin|Google_Search_Console
+ */
+
+/**
+ * Class WPSEO_Crawl_Issue_Fetch
+ */
+class WPSEO_Crawl_Issue_Fetch {
+
+	/**
+	 * All the categories
```

Comment:
I don't understand what you mean with this comment.

I don't see when `$issues` will be void.